lib/atlas/scaler.rb

```diff
@@ -50,5 +55,12 @@ def new_attributes
           },
       }
     end
+
+    def copy_etengine_data_files
+      FileUtils.cp_r(
+        %w( fce load_profiles network ).
```

Comment:
Perhaps this can be a static constant. I don't feel to strongly about it though.